interface/fax/fax_dispatch.php

```diff
@@ -200,8 +200,7 @@ function mergeTiffs() {
       }
 
       if (!$info_msg) {
-        // The following is cloned from contrib/forms/scanned_notes/new.php:
-        //
+// scanned notes must be installed, and does not natively exist.
```

Comment:
Without re-creating the scanned forms code, this does not exist and is a bad reference.  The fax tool here is probably legacy poop anyway.

Reply:
The version that bowen has is more generic and useful.  Perhaps we should pull that in to another P/R